Description: Add Debian-specific LDFLAGS into LOPT_BASE
 Part of hardening fixes
Author: Seb James <seb.james@sheffield.ac.uk>
Forwarded: No need to forward.
Last-Update: 2014-02-23

--- a/makefile.brahms
+++ b/makefile.brahms
@@ -292,7 +292,7 @@
 	INCLUDES='-I$(PATH_INC)'
 
 	# linker flags
-	LOPT_BASE='-L$(PATH_BIN)' '-L$(PATH_TMP)'
+	LOPT_BASE='$(LDFLAGS) -L$(PATH_BIN)' '-L$(PATH_TMP)'
 
 	# group data
 	COPT_COMPONENT='-I$(PATH_SML_NSP)' $(FLAG_SHARED)
